public/apps/configuration/panels/get-started.tsx

```diff
@@ -35,47 +35,48 @@ import { Action, ResourceType } from '../types';
 import { API_ENDPOINT_CACHE, DocLinks } from '../constants';
 import { ExternalLink, ExternalLinkButton } from '../utils/display-utils';
 
-const setOfSteps = [
-  {
-    title: 'Add backends',
-    children: (
-      <>
-        <EuiText size="s" color="subdued">
-          Add authentication<EuiCode>(authc)</EuiCode>and authorization<EuiCode>(authz)</EuiCode>
-          information to<EuiCode>plugins/opendistro_security/securityconfig/config.yml</EuiCode>.
-          The <EuiCode>authc</EuiCode> section contains the backends to check user credentials
-          against. The <EuiCode>authz</EuiCode>
-          section contains any backends to fetch external identities from. The most common example
-          of an external identity is an LDAP group.{' '}
-          <ExternalLink href={DocLinks.AuthenticationFlowDoc} />
-        </EuiText>
-
-        <EuiSpacer size="m" />
-
-        <EuiFlexGroup gutterSize="s" wrap>
-          <EuiFlexItem grow={false}>
-            <ExternalLinkButton
-              fill
-              href={DocLinks.BackendConfigurationDoc}
-              text="Create config.yml"
-            />
-          </EuiFlexItem>
-          <EuiFlexItem grow={false}>
-            <EuiButton
-              data-test-subj="review-authentication-and-authorization"
-              onClick={() => {
-                window.location.href = buildHashUrl(ResourceType.auth);
-              }}
-            >
-              Review authentication and authorization
-            </EuiButton>
-          </EuiFlexItem>
-        </EuiFlexGroup>
+const addBackendStep = {
+  title: 'Add backends',
+  children: (
+    <>
+      <EuiText size="s" color="subdued">
+        Add authentication<EuiCode>(authc)</EuiCode>and authorization<EuiCode>(authz)</EuiCode>
+        information to<EuiCode>plugins/opendistro_security/securityconfig/config.yml</EuiCode>. The
+        <EuiCode>authc</EuiCode> section contains the backends to check user credentials against.
+        The <EuiCode>authz</EuiCode>
+        section contains any backends to fetch external identities from. The most common example of
```

Comment:
Just confirm: is this "external identities" supposed to be replaced?

Reply:
I think my PR https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/619 has already done the replacement. Rebase to latest and solve conflict should work.

Reply:
Rebased to latest and resolved conflict. Now you should see backend roles.